src/joystick/joystick.cc

```diff
@@ -172,4 +174,47 @@ void Joystick::Close() {
   }
 }
 
+size_t Joystick::Remap(size_t num, size_t type){
+  if(mode_ == "Logitech_F710"){
+    // no need to remap
+  }
+  else if (mode_ == "Sony_DualShock_4"){
+    if(type == JS_EVENT_AXIS){
+      switch(num){
+        case 5:
+        case 3:
+          num--;
+          break;
+        case 2:
+        case 4:
+          num ++;
+          break;
+        default:
+          break;
+      }
+    }
+    else if (type == JS_EVENT_BUTTON){
+      switch(num){
+        case 2:
+        case 1:
```

Comment:
ditto here...would be nice to label what buttons these cases are for. Not necessary if you no longer have that info but it will be nice in the future

Reply:
I see, Ill add in the axis as comments